openverse_catalog/dags/maintenance/add_license_url.py

```diff
@@ -0,0 +1,179 @@
+"""
+# Add license URL
+
+Add license_url to all rows that have NULL in their meta_data fields.
+The license_url is constructed from the license and license_version fields.
+
+This is a maintenance DAG that should run once.
+"""
+import logging
+import os
+from datetime import datetime
+from textwrap import dedent
+from typing import Literal
+
+import jinja2
+from airflow.models import DAG
+from airflow.operators.python import BranchPythonOperator, PythonOperator
+from airflow.providers.postgres.hooks.postgres import PostgresHook
+from common import slack
+from common.constants import XCOM_PULL_TEMPLATE
+from common.licenses.constants import get_reverse_license_path_map
+from common.slack import send_message
+from psycopg2._json import Json
+
+
+DAG_ID = "add_license_url"
+DB_CONN_ID = os.getenv("OPENLEDGER_CONN_ID", "postgres_openledger_testing")
+
+ALERT_EMAIL_ADDRESSES = ""
+RETURN_ROW_COUNT = lambda c: c.rowcount  # noqa: E731
```

Comment:
Could we re-use the existing lambda we have for this?
https://github.com/WordPress/openverse-catalog/blob/af205795b412da9f3d0d7f6d7e066b7d039b1763/openverse_catalog/dags/common/loader/sql.py#L50